envoy/api/v2/core/http_uri.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package envoy.api.v2.core;
+
+import "validate/validate.proto";
+
+// Envoy external URI descriptor
+// [#not-implemented-hide:]
+message HttpUri {
+  // The HTTP server URI. It should be a full FQDN with protocol, host and path.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //    uri: https://www.googleapis.com/oauth2/v1/certs
+  //
+  string uri = 1 [(validate.rules).string.min_bytes = 1];
+
+  // Specify how `uri` is to be fetched. Today, this requires an explicit
+  // cluster, but in the future we may support dynamic cluster creation or
+  // inline DNS resolution. See `also
```

Comment:
This will just format as `See also.` with also as a link.

Reply:
Thanks.  Updated